### PR TITLE
Add talk proposal for February 2019

### PR DESCRIPTION
**Pull request by emodric**

### CODE REVIEW

**Comment by zoka123** [comment]
@emodric Thanks for the PR, it seems really interesting, I'd like to hear more about Layouts, as I don't know much except that it exists :)

I suggest you focus on the tech side of the journey, what were challenges and problems you had to solve, what did you learned along the way, and why you decided to build Layouts instead of using something that already exists (if a similar solution exists at all).

I believe it goes without saying, please don't make a sales pitch from your talk, target audience are devs :)

**Comment by emodric** [comment]
@ilukac is the lead guy here, so speak to him :)

File: meetups/2019/02/ivo-lukac-edi-modric--open-source-to-the-core [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Open source to the core
+
+- __Speaker__: Ivo Lukač & Edi Modrić
+- __Duration__: 25 min
+
+## Description
+
+After many years of opensourcing lot of smaller features in eZ and Symfony space, we now opensourced
+our flagship product: Netgen Layouts. In this talk we will share why and how you can start using it.
```
Comment (msvrtan) [inline meetups/2019/02/ivo-lukac-edi-modric--open-source-to-the-core]: Sorry guys, but title and description don't really go together... Title is about opensourcing while description is about how to use your product...

File: meetups/2019/02/ivo-lukac-edi-modric--open-source-to-the-core [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Open source to the core
+
+- __Speaker__: Ivo Lukač & Edi Modrić
```
Comment (msvrtan) [inline meetups/2019/02/ivo-lukac-edi-modric--open-source-to-the-core]: Having 2 speakers for a 25 minute slot feels too much... as Ivo has more experience talking and I feel Edi is much closer to the code part, I would suggest that Edi does it himself

**Comment by ilukac** [comment]
Fellow zgphp-ers,

It will not be any kind of sales pitch, we are planning to share why we open sourced the product (intro by me) and how we open sourced it (by Edi). 

As we drafted the talk summary in hurry, I see now it is not clear enough so I propose this:
"After many years of open-sourcing lot of smaller features in eZ and Symfony space, we now open-sourced our flagship product: Netgen Layouts. In this talk, we will share why we open sourced it and also how we did it, to encourage to similar efforts. "

Ok?

**Comment by msvrtan** [comment]
@ilukac 👍 Just update the description then!

**Comment by emodric** [comment]
@msvrtan Done!

**Comment by emodric** [comment]
@ilukac Do you think we should up the length of the talk?

**Comment by ilukac** [comment]
I think 25 minutes roughly will be ok, me 10 minutes, Edi 15 min

**Pull request merged**